pkg/cmd/release-payload-controller/payload_accepted_controller.go

```diff
@@ -133,9 +139,75 @@ func (c *PayloadAcceptedController) sync(ctx context.Context, key string) error
 		return err
 	}
 
+	originalCondition := conditions.GetReleasePayloadCondition(originalReleasePayload, v1alpha1.ConditionPayloadAccepted)
```

Comment:
make a function in library-go that takes []Conditions and returns you the *Condition